opsdroid/configuration/schema.yaml

```diff
@@ -0,0 +1,118 @@
+welcome-message: bool()
```

Comment:
This is optional

Reply:
added welcome-message: bool(required=False) and test  test_load_config_valid_without_wellcome_message